pkg/server/service.go

```diff
@@ -84,7 +84,16 @@ func (s *Server) CreateEntry(ctx context.Context, req *pb.CreateEntryRequest) (*
 			slog.Warn("failed validating hashedrekord request", "error", err.Error())
 			return nil, status.Errorf(codes.InvalidArgument, "invalid hashedrekord request")
 		}
-		serialized, err = protojson.Marshal(entry)
+		hrEntry := &pb.Entry{
```

Comment:
I have a slight preference to make this change to `ToLogEntry` for each type, like https://github.com/sigstore/rekor-tiles/pull/287/files#diff-2a03147d283232db2861d00d3281b65d506efae367704cbd41d981e03dfaa6beL34, rather than have the service be responsible for knowing the Kind and ApiVersion

Reply:
yeah that's definitely much nicer